src/NUnitFramework/testdata/TestCaseAttributeFixture.cs

```diff
@@ -96,7 +96,7 @@ public void MethodWithIncludePlatform(int num)
         [TestCase(2, ExcludePlatform = "Linux")]
         [TestCase(3, ExcludePlatform = "MacOSX")]
         [TestCase(4, ExcludePlatform = "XBox")]
-        public void MethodWitExcludePlatform(int num)
+        public void MethodWithExcludePlatform(int num)
```

Comment:
Note that this is also corrected in https://github.com/nunit/nunit/pull/2597/files#diff-e41bbd1635751322b84ea837982ac8bdR99

Reply:
Let's see who wins 😁